extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.tsx

```diff
@@ -211,7 +211,7 @@ function TrackedCornerstoneViewport(props) {
           seriesDescription: SeriesDescription,
           patientInformation: {
             patientName: PatientName
-              ? OHIF.utils.formatPN(PatientName.Alphabetic)
+              ? OHIF.utils.formatPN(PatientName)
```

Comment:
Let's no longer assume that the patient name is an object with an Alphabetic field.